packages/next/src/server/app-render/app-render.tsx

```diff
@@ -801,11 +801,14 @@ async function renderToHTMLOrFlightImpl(
     req.originalRequest.on('end', () => {
       const staticGenStore =
         ComponentMod.staticGenerationAsyncStorage.getStore()
+      const prerenderStore = prerenderAsyncStorage.getStore()
+      const isPPR = !!prerenderStore?.dynamicTracking?.dynamicAccesses?.length
```

Comment:
nit: maybe `isPPRFullyStatic` or something? Since I guess you could technically have PPR enabled but have no dynamic access

Reply:
Any usage of ppr don't want to show this one as we'll have separate tool for that